src/js/angular/core/services/cookie.service.js

```diff
@@ -0,0 +1,76 @@
+const COOKIE_NAME = '_wb=';
+const COOKIE_VERSION = 'WB1';
+const PATH = "; path=/";
+const EXPIRATION = "; expires=Fri, 31 Dec 2099 23:59:59 GMT";
+const EXPIRED = "; Max-Age=-99999999";
+const COOKIE_SEPARATOR = ';';
+
+angular.module('graphdb.framework.core.services.cookieService', [])
+    .service('CookieService', ['$document', CookieService]);
+
+/**
+ * Service to manage cookies for the application. It checks if a specific cookie exists,
+ * sets it if necessary, retrieves it, or deletes it when required.
+ *
+ * @param {Object} $document - AngularJS service to interact with the document object.
+ * @return {Object} - Returns an object with methods to get, set, and delete cookies.
+ */
+function CookieService($document) {
```

Comment:
Why don't you use the angular's $cookie service? You can still wrap this around it

Reply:
in version 1.3.8 it is quite limited from to functionality. That's why I wrote our generic cookie service.